agent-codeless/build.gradle

```diff
@@ -143,11 +143,13 @@ jar {
     dependsOn shadowJar2
     enabled = false
 }
-
+def logbackContribVersion = '0.1.5'
 dependencies {
     compile 'org.slf4j:slf4j-api:1.7.26'
     compile 'org.slf4j:jcl-over-slf4j:1.7.26' // this is needed to pick up logging from apache http client
     compile 'ch.qos.logback:logback-classic:1.2.3'
+    compile "ch.qos.logback.contrib:logback-json-classic:$logbackContribVersion"
```

Comment:
there used to be more dependencies which is why i declared a variable


...and I wanted to try out multi-line comments